compiler/rustc_error_codes/src/error_codes/E0780.md

```diff
@@ -0,0 +1,25 @@
+Cannot use `doc(inline)` with wildcard imports
+
+Erroneous code example:
+
+```compile_fail,E0780
+extern crate foo;
+
+#[doc(inline)] // error: invalid doc argument
+pub use foo::Foo as _;
+```
+
+When using a wildcard import the `doc` attribute currently only supports:
+
+ * hidden
+
+To fix this error either change to one of the available arguments or remove the
+`doc` attribute. 
+
+Example:
+
+```
+extern crate foo;
+
```

Comment:
````suggestion
```edition2018
````

Reply:
Hmm, actually it seems like you can't use doctests for multi-crate projects :/ https://github.com/rust-lang/rust/blame/353f3a38460d3c2222d2ab29143f48595b1a32a9/compiler/rustc_error_codes/src/error_codes/E0466.md#L16
So I guess just add `ignore` (and remove the `extern crate` still).